Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -49,20 +49,34 @@
 require "date"
 require "missing_formula"
 require "digest"
+require "cli_parser"
 
 module Homebrew
   module_function
 
   def audit
-    inject_dump_stats!(FormulaAuditor, /^audit_/) if ARGV.switch? "D"
+    args = Homebrew::CLI::Parser.new do
+      switch      "--strict",             "strict"
```

Comment:
I think it would be good if the second argument was inferred for all of these except `"Activates debugging and profiling"`.